pkg/kev/converter/kubernetes/helpers_test.go

```diff
@@ -37,7 +38,7 @@ func init() {
 
 func assertLog(level logrus.Level, message string, fields map[string]string) {
 	Expect(hook.LastEntry().Level).To(Equal(level))
-	Expect(hook.LastEntry().Message).To(Equal(message))
+	Expect(cmp.Diff(hook.LastEntry().Message, message)).To(BeEmpty())
```

Comment:
Curious on why you had to diff here?

Reply:
When I was investigating I couldn't see the whole sentence, just the small difference and it is not useful. This will basically show the 2 sentences in full so it';s easier to understand what changed